cayleypy/graphs_lib.py

```diff
@@ -92,6 +97,37 @@ def prepare_graph(name, n=0) -> CayleyGraph:
             generators.append(perm)
             generator_names.append("R" + str(prefix_len - 1))
         return CayleyGraph(generators, dest=list(range(n)), generator_names=generator_names)
+    elif name == "cubic_pancake":
+        assert n >= 2
+        subset = kwargs['S']
```

Comment:
What if 'S' is not in kwargs?

Do this: `assert 'S' in kwargs, "For cubic_pancake must specify parameter S."`